### PR TITLE
fix: reformat navigation bar and background

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
* updated the navigation backgroundimage
* updated the font color, font size and hovering color of navigation
image

* updated the font size , font color and overall body background 
image

* updated transparent logos
image

### CODE REVIEW

File: src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys-sphinx-theme-variable.css [hunk 1]
```diff
@@ -40,29 +40,26 @@
...
   --pst-font-size-h6: 11px;
-
-  /**
+  --pst-global-line-height: 24px;
+  --bs-nav-link-font-size: 14px;
+  --bs-navbar-color: var(--bs-ast-navbar-color) /**
 * table font sizes
 */
-  --pst-table-font-size: 16px;
+    --pst-table-font-size: 16px;
```
Comment (jorgepiloto) [inline src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys-sphinx-theme-variable.css:54]:
```suggestion
  --pst-table-font-size: 16px;
```

File: doc/source/conf.py [hunk 2]
```diff
@@ -13,7 +13,8 @@
 from ansys_sphinx_theme import (
     __version__,
     ansys_favicon,
-    ansys_logo_black,
+    ansys_logo_dark,
+    ansys_logo_light,
```
Comment (jorgepiloto) [inline doc/source/conf.py:17]: Let's go with the approach @klmcadams suggested by renaming these. Even if we break backwards support.

**Review by jorgepiloto — APPROVED** [review]
Thanks, @Revathyvenugopal162!

**Comment by Revathyvenugopal162** [comment]
@ansys/pyansys-core I have added the logo naming suggested by @klmcadams and will proceed with merging this PR.

File: doc/source/conf.py [hunk 3]
```diff
@@ -13,7 +13,8 @@
 from ansys_sphinx_theme import (
     __version__,
     ansys_favicon,
-    ansys_logo_black,
+    ansys_logo_dark_mode,
+    ansys_logo_light_mode,
     ansys_logo_white,
```
Comment (RobPasMue) [inline doc/source/conf.py:18]: Can we avoid breaking changes and add a deprecation warning to whomever is using the old images?

**Comment by RobPasMue** [comment]
Oh well - you already merged it @Revathyvenugopal162...

File: doc/source/conf.py (hunk 3 above)
Comment (Revathyvenugopal162) [reply doc/source/conf.py:18]: Sure, we can add docs and depreciation warning, i will add those in another PR. i thought of discuss it on our meeting (about logos and upcoming changes with sidebar)

**Comment by Revathyvenugopal162** [comment]
> Oh well - you already merged it @Revathyvenugopal162...

yeah, i will open an overall PR to add docs and warnings with logos and all upcoming changes.

**Pull request merged**